backend/handlers/handlers.go

```diff
@@ -882,7 +882,7 @@ func (handlers *Handlers) getExchangeMoonpayBuySupported(r *http.Request) (inter
 			break
 		}
 	}
-	return acct != nil && exchanges.IsMoonpaySupported(acct.Coin().Code()), nil
+	return acct != nil && !acct.Offline() && exchanges.IsMoonpaySupported(acct.Coin().Code()), nil
```

Comment:
Doesn't feel semantically correct. The endpoint is supposed to respond whether account's coin is supported. I don't believe you need an account to be "online" in order to get a valid response from `IsMoonpaySupported`.

Reply:
You don't have to have an online account to call `IsMoonpaySupported()`, but you need an online account to be able to use the account in moonpay. this is a quick fix / hack to not list offline accounts.

Reply:
Ok. Add a TODO on top? :pray:

Reply:
I have an idea for a general fix after this:

A higher order component that basically does what account.tsx already does: initialize an account, show spinner while syncing, show 'offline' if offline, etc. Factor that out of account.tsx, apply it there and in moonpay.tsx (and possibly even in the portfolio).

Will add the TODO